native.ahoy.yml

```diff
@@ -0,0 +1,32 @@
+ahoyapi: v2
+commands:
+  exec:
+    # These env variables come from https://github.com/wodby/docker4drupal/issues/273
+    cmd: bash -c "$@"
+  up:
+    # See https://laravel-guide.readthedocs.io/en/latest/valet/
+    cmd: valet up
```

Comment:
@weitzman I've installed Valet, but I'm not seeing the commands described in this file. What should these commands be?

Reply:
Fixed